### PR TITLE
Added template for case studies

### PR DESCRIPTION
**Pull request by atg-abhishek**
Addresses #205

### CODE REVIEW

**Review by srini1978 — CHANGES_REQUESTED** [review]
Do we have to add some information indicating PUE as well? Maybe in the energy efficiency section

File: Software_Carbon_Intensity/case_study_template.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+### Technical details of the components in the architecture
+
+_Textual description with technical details of each component provided in the architecture diagram_
+
+## Sites for Software Sustainability Actions
+
+_For each of the following sub-sections, indicate **where** and **how** actions can be taken in the following categories_:
+
+### Energy Efficiency
```
Comment (jawache) [inline Software_Carbon_Intensity/case_study_template.md:27]: In this section are you asking people to write out some ideas for how they might be able to reduce their score?

File: Software_Carbon_Intensity/case_study_template.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+### Carbon Awareness
+
+1. _site of action_
+2. _description of the action_
+
+## Procedure
+
+### (What) Software boundary
+
+_Describe the components that are included in the software systems along with those that have been excluded with **reasons for exclusion**_
```
Comment (jawache) [inline Software_Carbon_Intensity/case_study_template.md:46]:
```suggestion
_Describe the components that are included in the software systems, if any major components are not included then please provide **reasons for exclusion**_.

For example:
- Front end mobile application
- Network traffic between client mobile applications and servers
- Network traffic between servers and database
- Back-end servers.
- Databases
- Test infrastructure
```

File: Software_Carbon_Intensity/case_study_template.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+### (Scale) Functional unit 
+
+_Describe the functional unit that best controls the scaling of the software system_
```
Comment (jawache) [inline Software_Carbon_Intensity/case_study_template.md:50]:
```suggestion
_Describe the [functional unit](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#functional-unit-r) that best controls the scaling of the software system.

The choice of functional unit applies to all components in your software boundary. 

For example:
- per API Call
- per User
_
```

File: Software_Carbon_Intensity/case_study_template.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+### (How) Quantification method
+
+_Describe how you are applying either a lab-based measurement based on models or calculating the a rate, an SCI value, for every software component of the system_
```
Comment (jawache) [inline Software_Carbon_Intensity/case_study_template.md:54]:
```suggestion
_For each software component in your software boundary, decide whether you are going to **measure** using real-world data or **calculate** an estimate via models, provide a reason and any useful details for each choice.

For example:
- Front-end mobile application. 
 - Calculate an estimate using a model that represents a single user using the application. 
 - We do not have the right to export real-world measurements from individuals mobiles.
 
- Back-end servers
 - Calculate an estimate of the energy consumption using models which take as input the CPU utilization of servers.
 - Calculate an estimate of the embodied carbon using the Treads low-resolution models based on AWS data.
```

**Comment by jawache** [comment]
Added some comments and suggestions, I thought some examples might help people grasp the ask better :)

File: Software_Carbon_Intensity/case_study_template.md (hunk 1 above)
Comment (atg-abhishek) [reply Software_Carbon_Intensity/case_study_template.md:27]: So asking for more details on specific things that they are doing to reduce the score by indicating the "where" and "how" of the actions

**Comment by atg-abhishek** [comment]
> Do we have to add some information indicating PUE as well? Maybe in the energy efficiency section

We haven't settled on using PUE yet right?

File: Software_Carbon_Intensity/case_study_template.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+- Front end mobile application
+- Network traffic between client mobile applications and servers
+- Network traffic between servers and database
+- Back-end servers.
+- Databases
+- Test infrastructure
+
+### (Scale) Functional unit 
+
+_Describe the [functional unit](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#functional-unit-r) that best controls the scaling of the software system.
```
Comment (atg-abhishek) [inline Software_Carbon_Intensity/case_study_template.md:58]:
```suggestion
_Describe the [functional unit](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#functional-unit-r) that best controls the scaling of the software system._
```

**Comment by atg-abhishek** [comment]
> Added some comments and suggestions, I thought some examples might help people grasp the ask better :)

Thanks @jawache! Much appreciated!

**Comment by Henry-WattTime** [comment]
Approved by working group.

**Pull request merged**